gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -0,0 +1,79 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_replication'
+require 'util/postgres_admin'
+
+module ApplianceConsole
+  class DatabaseReplicationStandby < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr standby register'.freeze
+
+    def initialize
+      set_defaults
+    end
+
+    def set_defaults
+      self.cluster_name      = "replication_cluster"
+      self.node_number       = nil
+      self.database_name     = "vmdb_production"
+      self.database_user     = "root"
+      self.database_password = nil
+      self.primary_host      = nil
+      self.standby_host      = LinuxAdmin::NetworkInterface.new(NETWORK_INTERFACE).address
+    end
+
+    def ask_questions
+      clear_screen
+      say("Establish Replication Standby Server\n")
+      ask_for_cluster_name
+      ask_for_node_number
+      ask_for_database_credentials
+      ask_for_standby_host
+      ask_continue_if_already_configured? &&
+        confirm(:including_standby_host)
+    end
+
+    def ask_for_standby_host
+      ask_for_ip_or_hostname("Standby Server hostname or IP address", standby_host)
+    end
+
+    def activate
+      say("Configuring Replication Standby Server...")
+      confirm_no_standby_database? &&
+        create_config_file(standby_host) &&
+        clone_standby_server &&
+        start_postgres &&
+        register_standby_server
+    end
+
+    def data_dir_empty?
```

Comment:
It doesn't look like this is being called anywhere ... Or am I missing something?